### PR TITLE
Script to add and delete Network Virtualisation interfaces

### PR DESCRIPTION
**Pull request by hathyaga**
This script will add and delete Network virtualized interfaces
using the SRIOV capable IO adapter given as an input by the user
and performs this operation multiple times based on the count
given by the user as well

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by hathyaga** [comment]
[job.log.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1818125/job.log.txt)

Job log

File: io/net/Network_virtualization_dlpar.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        return "Command '%s' exited with %d.\nOutput:\n%s" \
+               % (self.command, self.exitcode, self.output)
+
+
+class NetworkVirtualization(Test):
+    '''
+    Adding and deleting Network Virtualized ports from the vios
+    '''
+    @skipUnless("ppc" in distro.detect().arch,
+                "supported only on Power platform")
```
Comment (harish-24) [inline io/net/Network_virtualization_dlpar.py]: Please add skipIf only once here. There is no need for adding it before `test()` function or any other function as we do not want to execute the `setup()` if test needs to be skipped.
```
@skipIf(IS_POWER_NV or IS_KVM_GUEST, 
        "This test is not supported on KVM guest or PowerNV platform")
```

Comment (harish-24) [reply io/net/Network_virtualization_dlpar.py]: Note that Skip message in the above comment is moved to next line as the previous implementation seemed to have caused unnecessary spaces.
```
 TestSkipError: This test is not supported on KVM         guest or PowerNV platform
```

File: io/net/Network_virtualization_dlpar.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        except CmdError as details:
+            self.log.debug(str(details))
+            self.fail("Starting service %s failed", svc)
+
+        output = process.system_output("lssrc -a", ignore_status=True,
+                                       shell=True, sudo=True)
+        if "inoperative" in output:
+            self.fail("Failed to start the rsct and rsct_rm services")
+
+        for val in range(self.count):
```
Comment (harish-24) [inline io/net/Network_virtualization_dlpar.py]: If `val` is not used, we may replace as follows
```
for _ in range(self.count):
```

**Comment by hathyaga** [comment]
New PR: https://github.com/avocado-framework-tests/avocado-misc-tests/pull/826

**Pull request closed without merging**